auto_joiner.py

```diff
@@ -617,7 +617,12 @@ def main():
 
     print("Waiting for correct page...", end='')
     if wait_until_found("#teams-app-bar", 60 * 5) is None:
-        exit(1)
+        # click the Try again button if teams load error
+        try_again = wait_until_found("button[class*='ts-btn ts-btn-primary oops-button']", 10)
+        if try_again is not None:
+            try_again.click()
```

Comment:
After clicking the retry button it would be better to wait for the "#teams-app-bar" again. Otherwise the code after this would start running while it's still loading. Please add this.

Reply:
added for loop functions to try to get #teams-app-bar element status to decide the next step